src/Package/ComparePackageRepo.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace SiteStudio\Package;
+
+use Symfony\Component\Yaml\Yaml;
+use SiteStudio\Package\ArrayDiff;
+use SiteStudio\Package\ConfigInitializer;
+
+class ComparePackageRepo extends ConfigInitializer {
+
+    /**
+     * ConfigInitializer constructor.
+     */
+    public function __construct() {
+        parent::__construct();
+        $this->yamlDiffReport();
+    }
+
+    /**
+     * Yaml Difference Report.
+     */
+    public function yamlDiffReport() {
+        $create_directory = "mkdir -p $this->comparisionFolder";
+        shell_exec($create_directory);
+
+        $this->accessToken = trim(readline('Please Enter Private Access token for Package Management Repo'));
+        $list_of_projects_cmd = "curl --header 'PRIVATE-TOKEN: $this->accessToken' '$this->gitlabApiPath/groups/$this->packageRepoGroup/projects?per_page=500' >> ./$this->comparisionFolder/$this->projectsPath";
```

Comment:
@rakshithtb this is hard wired to Gitlabs, I'm going to merge to unblock testing, but create an Issue to abstract this as the library should allow other sources e.g. github, OR namespace the functions so it's clear these classes only are intended to support Gitlabs.